src/shared/lib/StoreUtils.ts

```diff
@@ -1109,33 +1109,16 @@ export function findUncalledMutationMolecularProfileId(
 }
 
 export function findMrnaRankMolecularProfileId(
-    molecularProfilesInStudy: MobxPromise<MolecularProfile[]>
+    molecularProfilesInStudy: string[]
 ) {
-    if (!molecularProfilesInStudy.result) {
-        return null;
-    }
-
-    const regex1 = /^.+rna_seq.*_zscores$/; // We prefer profiles that look like this
-    const regex2 = /^.*_zscores$/; // If none of the above are available, we'll look for ones like this
-    const preferredProfile:
-        | MolecularProfile
-        | undefined = molecularProfilesInStudy.result.find(
-        (gp: MolecularProfile) =>
-            regex1.test(gp.molecularProfileId.toLowerCase())
+    const regex1 = /^.+rna_seq.*_zscores$/i; // We prefer profiles that look like this
```

Comment:
@inodb this will still allow non mrna profiles when the column header implies mrna.  However, if  i change it to mrna_seq (is that valid?) then we no longer match most of the profiles currently used.  So we'd have to change the matching to make it consistent.

Reply:
This looks good to me